.changes/js-submenu-in-options.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tauri-apps/api": "patch:bug"
+---
+
+Fix submenu created as a menu item instead of a submenu when created by using an object in the `items` field in the options object pssed to `Menu.new` or `Submenu.new`.
```

Comment:
```suggestion
Fix submenu created as a menu item instead of a submenu when created by using an object in the `items` field in the options object passed to `Menu.new` or `Submenu.new`.
```